### DISCUSSION TITLE
Jupyter lab failed to run.

### DISCUSSION THREAD

**Discussion by Geozike** [giswqs/geog-312#68] [Q&A] [answered]
Sir, my jupyter lab is not running  because it is being used by another process At line:1 char:1
![problem 5](https://github.com/user-attachments/assets/ce292fc3-9aab-4b76-8eae-2c21dd8f4bc3)

**Comment by giswqs** [comment, marked as answer]
See https://youtu.be/-YnfmHBHkWQ
